### PR TITLE
Added dependencies and python path assignment

### PR DESCRIPTION
**Pull request by andrewginns**
Added dependencies for the python example

### CODE REVIEW

**Comment by andrewginns** [comment]
Broken links took a couple tries to fix. Will now link to the top level README in the master branch.

File: python/README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Python example for DNB Open Banking APIs
 
+## Requirements
+
+* Python 2.7
+* pipenv - pip install pipenv
+* requests - pip install requests
```
Comment (relekang) [inline python/README.md]: Installing dependencies globally is not the best idea, because different projects might depend on different versions, and not something we should encourage. This is why we use pipenv.

File: python/README.md [hunk 2]
```diff
@@ -19,11 +25,10 @@ After adding the credentials you can run the example with
 
 ```shell
 pipenv install # install dependencies
+pipenv --python path/to/python/2.7/exe
```
Comment (relekang) [inline python/README.md]: Why is this necessary?

**Comment by andrewginns** [comment]
Tried in Ubuntu. Double checked my API keys to ensure they are correct. I get a token error.

pipenv run python getting_started.py
Traceback (most recent call last):
  File "getting_started.py", line 86, in <module>
    main()
  File "getting_started.py", line 39, in main
    api_token = api_token_response.json()["tokenInfo"][0]["jwtToken"]
KeyError: 'tokenInfo'
